opflexagent/gbp_ovs_agent.py

```diff
@@ -112,6 +123,20 @@ def __init__(self, root_helper=None, **kwargs):
         # TODO(ivar): make this component pluggable.
         self.ep_manager = ep_manager.EndpointFileManager().initialize(
             self.host, self.bridge_manager, kwargs)
+        self.init_firewall()
+
+    def _invoke_ep_mgr(self, port):
```

Comment:
That was my original  implementation. However, it requires a user-configurable parameter. I received feedback earlier that we should try to avoid new configuration parameters if possible, and keep a single implementation that supports multiple things. I'm certainly open to revisiting this, but wanted to provide context for the current implementation.